DateTimePicker/DateTimePicker.js

```diff
@@ -49,20 +30,16 @@ const PickerCell = kind({
  */
 class DateTimePickerBase extends React.Component {
 	static propTypes = /** @lends agate/DateTimePicker.DateTimePickerBase.prototype */ {
-		onChange: PropTypes.func,
-		onClose: PropTypes.func,
-		onSave: PropTypes.func
+		onChange: PropTypes.func
 	};
 
 	constructor (props) {
 		super(props);
+
 		this.state = {
-			hour: '12',
-			minute: '00',
-			meridiem: $L('AM'),
-			month: '1',
-			day: '1',
-			year: '2018'
+			date: new Date(),
+			time: new Date(),
+			dateTime: new Date()
```

Comment:
Do you explain why this component has these states, please?

Reply:
The existing DateTimePicker was built using state and also had a "save" button. I was not sure why that was necessary, as the colleague that worked on it was on holiday.